core/imagedownloader.cpp

```diff
@@ -85,26 +85,31 @@ static bool hasVideoFileExtension(const QString &filename)
 #ifdef LIBRAW_SUPPORT
 QImage fetchRawThumbnail(const QString &filename)
 {
-	LibRaw raw; // Might think about reusing that, one instance per thread
-
-	// TODO: Convert filename to UTF-16 for windows
-	if (raw.open_file(qPrintable(filename)) != LIBRAW_SUCCESS ||
-	    raw.unpack_thumb() != LIBRAW_SUCCESS) {
+	// Might think about reusing that, one instance per thread
+	// This is quite heavy, so better not allocate it on the stack
+	LibRaw *raw =  new LibRaw;
```

Comment:
Please don't add raw owning pointers.

Turn this line into
```
        auto raw = std:::make_unique<LibRaw>()
```
and remove all the `delete raw;` instances.

This might require an `#include <memory>`.

Reply:
You removed the UTF-16 comment. Was that on purpose?